person.rb

```diff
@@ -0,0 +1,30 @@
+require 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rental
```

Comment:
- [ ] Getters and Setters for `@rentals` should be present which means the `rental` should be `attr_accessor`, kindly fix that 🙃 😉